### PR TITLE
like/dislike button added

### PR DESCRIPTION
**Pull request by yaroslavponomaryov**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -31,4 +31,28 @@
     list-style: none;
     padding-left: 0;
 
+}
+
+.like {
+    border: blue;
+    border-radius: 8px 0px 0px 8px;
```
Comment (sirKevlar) [inline src/index.css:38]: Styling comment:
When we navigate to all articles from the burger activated dropdown menu, I would expect the dropdown menu to close, whereas at the moment it stays open

File: src/components/SingleArticle.jsx [hunk 2]
```diff
@@ -24,6 +25,11 @@ const SingleArticle = () => {
         })
     }, [])
 
+    const handleClick = (vote) => {
+        updateArticleVotes(article_id, vote)
+        setArticle(articleVotesChanger(article, vote))
```
Comment (sirKevlar) [inline src/components/SingleArticle.jsx:30]: Technically, this line of code should execute before the patch for this to be truly optimistic rendering

File: src/components/SingleArticle.jsx [hunk 3]
```diff
@@ -24,6 +25,11 @@ const SingleArticle = () => {
         })
     }, [])
 
+    const handleClick = (vote) => {
+        updateArticleVotes(article_id, vote)
```
Comment (sirKevlar) [inline src/components/SingleArticle.jsx:29]: This patch could fail. In order to complete this ticket, you need to provide some client feedback on failure and also undo the optimistic rendering

File: src/index.css (hunk 1 above)
Comment (yaroslavponomaryov) [reply src/index.css:38]: I've just tested it, and it's closing in my browser:( I don't know, what's the problem
I'll try to solve it

**Review by xViolaine — APPROVED** [review]

**Pull request merged**